### PR TITLE
Adding a hyperlink for two references in the README.md

### LINKED ISSUE #47131: please add CodePen link to
### Describe the Issue

In the amazing doc of README.md, the CodePen website doesn't have a link.
When a great website, like a CodePen, doesn't have a link in a README.md of a quality OpenSource project,
a small kitten starts crying somewhere. A small, shaking, frozen kitten.

Please think of kittens and add a hyperlink to the CodePen.
Edited by moderator

Special thanks to [@eazyangel](https://twitter.com/eazyangel)  who made this ticket happen, as we have realized there are no open "first timers only" tickets available.

### Affected Page

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/README.md

### Steps to Reproduce

Open the README.md and try clicking on CodePen. Nothing happens, but ideally, it should transfer to the CodePen's webpage.

### Recommended fix or suggestions

Make CodePen a hyperlink, pointing at the correct web page.

### Screenshots

image

### PR DESCRIPTION
**Pull request by DylanDevelops**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #47131

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ freeCodeCamp.org offers several free developer certifications. Each of these cer
 
 Each of these 50 projects in the freeCodeCamp.org curriculum has its own agile user stories and automated tests. These help you build up your project incrementally and ensure you've fulfilled all the user stories before you submit it.
 
-You can pull in these test suites through [freeCodeCamp's CDN](https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js). This means you can build these projects on websites like CodePen and Replit - or even on your local computer's development environment.
+You can pull in these test suites through [freeCodeCamp's CDN](https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js). This means you can build these projects on websites like [CodePen](https://codepen.io) and [Replit](https://replit.com) - or even on your local computer's development environment.
```
Comment (Sboonny) [inline README.md:31]:
```suggestion
You can pull in these test suites through [freeCodeCamp's CDN](https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js). This means you can build these projects on your cloud development environment of your choice  - or on your local computer's development environment.
```

**Review by Sboonny — CHANGES_REQUESTED** [review]
Currently, we want to remove external site, so people don't think they are a part of freeCodeCamp.

That's why you will find online editor, when you try to take new RWD projects for certifications.

**Comment by raisedadead** [comment]
@DylanDevelops thanks for taking the time to create the PR. We appreciate your hard work but we are going to have to close it. You are welcome to work on any of the other outstanding issues that are tagged "help wanted".

**Comment by DylanDevelops** [comment]
Thanks, I totally get that you don't wanna send people to places that aren't guaranteed safety like freecodecamp.

**Pull request closed without merging**